app/stats.go

```diff
@@ -5,10 +5,10 @@ import (
 	"github.com/codegangsta/cli"
 )
 
-func LsStats() cli.Command {
+func LsJournal() cli.Command {
 	return cli.Command{
-		Name:      "ls-stats",
-		ShortName: "stats",
+		Name:      "ls-journal",
```

Comment:
Are we going to have any other operation on journal? Otherwise I think `ls-` prefix is not necessary.